systemtest/src/test/java/io/strimzi/systemtest/operators/user/UserST.java

```diff
@@ -340,123 +344,100 @@ void testCreatingUsersWithSecretPrefix(ExtensionContext extensionContext) {
     }
 
     @ParallelNamespaceTest
-    void testTlsExternalUser(ExtensionContext extensionContext) {
+    void testTlsExternalUser(ExtensionContext extensionContext) throws IOException, InterruptedException {
         final TestStorage testStorage = storageMap.get(extensionContext);
-        final String namespaceName = StUtils.getNamespaceBasedOnRbac(Environment.TEST_SUITE_NAMESPACE, extensionContext);
-        final String clusterName = testStorage.getClusterName();
-        final String userName = testStorage.getKafkaUsername();
-        final String topicName = testStorage.getTopicName();
-
-        final AclRule aclRule = new AclRuleBuilder()
-            .withNewAclRuleTopicResource()
-                .withName(topicName)
-            .endAclRuleTopicResource()
-            .withOperations(AclOperation.WRITE, AclOperation.DESCRIBE)
-            .build();
+        String consumerGroupName = ClientUtils.generateRandomConsumerGroup();
```

Comment:
Don't we generate a consumer group in the TestStorage class?

Reply:
Unfortunately no, we only get consumerName.